### ISSUE TITLE
disability terminology - "visual" and directly-related terms

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-intro-accessibility#78]
_Note: This issue title includes "directly-related terms" to keep this issue going on tangents. In scope of this issue is the terminology quoted below, including "auditory". Feel free to open separate issues for other terms._

WAI resources such as [Introduction to Web Accessibility](https://www.w3.org/WAI/fundamentals/accessibility-intro/) and [How People with Disabilities use the Web - draft revision](https://deploy-preview-113--wai-people-use-web.netlify.app/people-use-web/abilities-barriers/) use the following categories of disabilities or impairments:
> auditory, cognitive and learning, neurological, physical, speech, visual

WCAG 2 includes:
> Accessibility involves a wide range of disabilities, including visual, auditory, physical, speech, cognitive, language, learning, and neurological disabilities.

Adam Cooper brought up an issue with "visual disabilities|impairments" versus "vision disabilities|impairments" in an [e-mail to wai-eo-editors](https://lists.w3.org/Archives/Public/wai-eo-editors/2023May/0003.html) that includes:
> The word  'visual' is about what is seen as in 'visual art'.
> The word 'vision' is about seeing rather than what is seen as in
'envisioning'. 
> to say something is 'visually impaired', therefore, is tantamount to saying
the appearance of that thing is itself impaired.
> The phrase 'vision impairment' rather than 'visually impaired' or 'visual
disability' is used by the World Health Organisation in the International
Classification framework of Functioning, Disability, and Health, and by the
U.N. more broadly.

(and [Kevin chimed in with a story](https://lists.w3.org/Archives/Public/wai-eo-editors/2023May/0004.html) and:
> I tend to use 'sight impaired' now.

Please add to this issue input &mdash; especially authoritative references &mdash; to inform the possibility of WAI changing terminology from "visual disability|impairment" to "vision disability|impairment" (or "sight disability|impairment").

**Comment by someone-took-my-username-already** [comment]
Hi Shawn ... thanks so much for following this up. What are the 'categories of disability' listed above based on?   

Given that the focus of W3 is technology and not health/disability, would it make sense for it to adopt a global framework for defining and describing disability, impairment, etc,?

The International Classification of Functioning, Disability, and Impairment, is the World Health Organisation's taxonomy (1500+ categories) which is used by governments across the globe (including the Australian government) to - inter alia - assess    levels of functioning for an individual within a BioPsychoSocial context.

There's no "approved list" of appropriate termrs of disability per se, but 'vision impairment' is the term used rather than 'visual disability'. 

The reasons for 'vision impairment' are integral to the structure of the ICF itself, but 'vision' is a body function and 'impairment' is defined such that - in my limited understanding at least - 'visual disability' makes little sense in the context of the ICF.

I guess the question is: "are the W3 specifications based in and on globally recognised ways of defining and describing disability/impairment/functioning/health conditions etc."?

The key for me is 'globally' ...

**Comment by shawna-slh** [comment]
Yes, W3C considers terminology from a global perspective.

**Comment by shawna-slh** [comment]
[EOWG discussed on 2 June 2023](https://www.w3.org/2023/06/02-eo-minutes.html#item04) &mdash; apologies for the missing minutes details.

Participants agreed that it is difficult to come to consensus on the best terminology around the world.

A primary question is: How important is this issue for people with visual/vision impairments?

The small number of participants at that EOWG meeting felt that the potential terminology change from "visual" to "vision" is not a high priority at this point.

We are leaving this issue open for additional input, and it is listed  [For anyone, Work for this Week](https://www.w3.org/WAI/EO/wiki/EOWG_Meetings#Work_for_this_week).

**Comment by someone-took-my-username-already** [comment]
so, as above, what is the standard or document or thinking or process that makes the language that W3 uses truly global?

With respect, a poll among five or so people on a conference call is hardly due consideration of this issue (especially when the minutes reflect more time was spent discussing the content of headings).

The point is broader than vision versus visual - it is about how disability-informed WAI really is ...

**Comment by shawna-slh** [comment]
> a poll among five or so people on a conference call 

agree. That is why we are leaving the issue open and soliciting more input.

In addition to the broad invitation to anyone, we have specifically asked people with visual/vision disabilities within the WAI community to comment.

**Comment by mraccess77** [comment]
I would add that not all vision/visual disabilities are related to the eye or "vision" - Cortical or cerebral Visual Impairment (CVI) is diagnosed when people show abnormal visual responses that aren't caused by the eyes themselves. https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/cerebral-visual-impairment-cvi

**Comment by tome-wilson** [comment]
I agree with 100% with https://github.com/w3c/wai-intro-accessibility/issues/78#issuecomment-1573367477

“Vision” indicates the sense of sight whereas “visual disability” could be mistranslated as “a disability that can be seen.”

**Comment by someone-took-my-username-already** [comment]
> I would add that not all vision/visual disabilities are related to the eye or "vision" - Cortical or cerebral Visual Impairment (CVI) is diagnosed when people show abnormal visual responses that aren't caused by the eyes themselves. https://www.nei.nih.gov/learn-about-eye-health/eye-conditions-and-diseases/cerebral-visual-impairment-cvi

yes, absolutely agree ...  'vision' is not a function of the eye in an of itself as phenomena like Charles Bonnet syndrome  or those  mentioned above ... which is why I said originally that 'vision is about seeing' and not 'vision is about the eye'.

**Comment by slewth** [comment]
This is a difficult subject. Particularly given the issue raised that 'visual disability' could imply a disability that is seen, as opposed to experienced. However, as others have said there are disabilities that relate to vision but are not about sight/eye health. Importantly I'd like to add that these can be missed by some standard impairment classifications. For example, specific light sensitivities.

In this respect, I'd support 'visual' which for me implies a broader impairment group than 'vision'. However, this may not be read/translated in this way by broader groups. As such, which ever the group agrees, I'd suggest this is defined in our materials and signposted if possible.

**Comment by someone-took-my-username-already** [comment]
> This is a difficult subject. Particularly given the issue raised that 'visual disability' could imply a disability that is seen, as opposed to experienced. However, as others have said there are disabilities that relate to vision but are not about sight/eye health. Importantly I'd like to add that these can be missed by some standard impairment classifications. For example, specific light sensitivities.
> 
> In this respect, I'd support 'visual' which for me implies a broader impairment group than 'vision'. However, this may not be read/translated in this way by broader groups. As such, which ever the group agrees, I'd suggest this is defined in our materials and signposted if possible.

1. difficult doesn't mean insoluble
2. I don't understand 'disability seen versus experienced' - forgive me if this is an issue of translation
3. what are the 'standard impairment classifications' you mention?
4. this is not about this or that group preferring one or the other - it's about WAI becoming disability-informed

**Comment by Sylvie42** [comment]
Visual disabilitiy or vision disability is not a priority for me. I am ok with visual disability.
